src/main/java/duke/Utils/TaskList.java

```diff
@@ -20,6 +20,7 @@ enum Type {
         DEADLINE("deadline"),
         EVENT("event"),
         DELETE("delete"),
+        FIND("find"),
```

Comment:
Great that you have implemented enums to keep track of the long list of keywords!